### PR TITLE
Fix some issues by gosec scann

### PR DESCRIPTION
**Pull request by m14815**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Fix some gosec G307 issues
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
First time for try some contribution 
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by m14815** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

Please check the CLI status again

**Comment by fedebongio** [comment]
/assign @wenjiaswe 
Can you take a look to the etcd part? Thanks!
/triage accepted
/remove-sig api-machinery

File: cluster/images/etcd/cp/cp.go [hunk 1]
```diff
@@ -32,7 +32,11 @@ func main() {
 	if err != nil {
 		log.Fatalf("unable to open source file [%s]: %q", os.Args[1], err)
 	}
-	defer sf.Close()
+	defer func() {
+		if err := sf.Close(); err != nil {
+			log.Printf("Error while closing sorce file; Error %v", err)
```
Comment (wenjiaswe) [inline cluster/images/etcd/cp/cp.go:37]: s/sorce/source

**Review by wenjiaswe — COMMENTED** [review]
/lgtm for cluster/images/etcd/cp/cp.go file with a spelling nit.

**Pull request closed without merging**